static/css/components/sort-dropper.less

```
@@ -0,0 +1,121 @@
+.sort-dropper {
+  display: inline-flex;
+}
+
+.sort-content {
+  display: flex;
+  background: @white;
+}
+
+.sort-content-inner {
+  display: flex;
+  flex-direction: column;
+  position: absolute;
+  margin-top: 10px;
+  background-color: @lightest-grey;
+  border: 1px solid;
+  border-radius: 5px;
+  border-color: @light-grey;
+  z-index: @z-index-level-1;
+
+  a {
+    text-decoration: none;
+    color: @grey;
+    padding-right: 25px;
+    border: 2px solid transparent;
+
+    &:link {
+      color: @black;
+    }
+
+    &:visited {
+      color: @black;
+    }
+
```

Small edit for concision :
```suggestion
&:link,
&:visited {
color: @black
```